helm/easeprobe/Chart.yaml

```diff
@@ -3,7 +3,7 @@ annotations:
 apiVersion: v2
 appVersion: "2.0.0"
 name: easeprobe
-version: "0.0.1"
+version: "1.0.0"
```

Comment:
It's a good practice to update the 'appVersion' field in the Chart.yaml file when you bump the chart version. [medium]